skcosmo/preprocessing/flexible_scaler.py

```diff
@@ -114,15 +114,9 @@ class KernelFlexibleCenterer(TransformerMixin, BaseEstimator):
     but with additional parameters, relative to which centering
     is carried out:
 
-
-    :param ref_cmean: an array with means for each column.
-    :type ref_cmean: array of shape (1, n_features)
-
-    :param ref_mean: an average for the whole kernel matrix
-    :type ref_mean: array
     """
 
-    def __init__(self, ref_cmean=None, ref_mean=None):
+    def __init__(self):
```

Comment:
No, you need to keep the old `__init__.` It makes checking everything much easier.

Reply:
You _don't_ initialize the scaler with the fitting parameters, you fit it with them. Just like when setting up a LR or PCA, you don't initialize with the fitting parameters.